### ISSUE TITLE
Example Compile Error in Mac OS Mojave

### ISSUE THREAD

**Issue by shbang91** [dartsim/dart#1200]
Hello,
I am working on Mac OS Mojave with DART 6.5.0
I just go to the
```
examples/atlasSimbicon
mkdir build & cd build
cmake .. & make
```
and getting the following error
```
ld: library not found for -lconsole_bridge
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [atlasSimbicon] Error 1
make[1]: *** [CMakeFiles/atlasSimbicon.dir/all] Error 2
make: *** [all] Error 2
```
I think this issue is related to the version of Mac OS.
Do you have a plan to fix this?

**Comment by shbang91** [comment]
Thanks a lot!

**Comment by scpeters** [comment]
actually, this looks related to https://github.com/ros/console_bridge/issues/52

**Comment by scpeters** [comment]
I think this will be fixed by https://github.com/ros/console_bridge/pull/60

**Comment by jslee02** [comment]
@scpeters Thanks for the fix! However, the fix didn't work for me. I think it fails to load `console_bridge` (and even `console_bridge::console_bridge`) because the target is not imported by just loading `DARTConfig.cmake`. Instead, `DARTConfig.cmake` only store the target name in `DART_LIBRARIES` without importing the target actually. I hope #1209 fixes this issue.

**Comment by scpeters** [comment]
Thanks @jslee02; I thought the target namespace was the problem, but I guess it's an issue in dart. Yes, #1209 seems to fix it.

**Comment by jslee02** [comment]
@scpeters Thanks for the confirmation. I think I learned a lot about target including target namespace!

Closing this issue as it's resolved. @shbang91 Please let us know if this is still persistent for you.

**Comment by scpeters** [comment]
It's resolved on the `release-6.7` branch, but it won't be fully resolved for downstream users until it's released in homebrew. Do you have a planned date for the 6.7 release?

**Comment by jslee02** [comment]
Sure. I plan to release 6.7 once #1212 is merged.
